packages/ButtonLink/ButtonLink.jsx

```diff
@@ -97,6 +111,7 @@ ButtonLink.defaultProps = {
   reactRouterLinkComponent: null,
   to: null,
   href: null,
+  fullwidth: null,
```

Comment:
Default should be `false`

Reply:
Hi @theetrain,
1 - I have added comment in buttonLink component. 
2 -I have added prop as on option to make the button full width for core-button component.